roles/senior_software_engineer_2.md

```diff
@@ -1,15 +1,17 @@
 # Senior Software Engineer 2
 
-Manchester & London, UK. Often advertised as Lead Software Engineer.
+(Often advertised as Lead Software Engineer.)
 
-Our Senior Engineer 2s turn our customer’s needs into roadmaps, coach teams to deliver, are cutting the best code of their career, still learn every day, and constantly strive to be nice humans.
+Our Senior Engineer 2s are Tech Leads, Architects, Thought Leaders, Full Stack Engineers, and Consultants. They are our most senior technologist within a customer account representing Made Tech and act as a hands-on CTO across one or more software delivery teams. They combine technical excellence, drive to deliver, and coaching, to achieve outcomes for our customers and their users, and to establish strong engineering cultures within our customers organisations. They find themselves working on a variety of different problems from monoliths to microservices, upskilling colleagues and customers, always finding themselves learning from others, while constantly striving to be nice humans :)
```

Comment:
I think more emphasis on outcomes would be good here, IMO we're pretty good at measuring Senior 2s by how they are achieving business outcomes as opposed to by what specific tasks they are carrying out.